sdk/spring/spring-cloud-azure-core/src/main/java/com/azure/spring/core/factory/AzureCredentialBuilderFactory.java

```diff
@@ -69,12 +79,20 @@ protected AzureProperties getAzureProperties() {
 
     @Override
     protected void configureService(T builder) {
+        final ProxyAware.Proxy proxy = getAzureProperties().getProxy();
```

Comment:
we should rewrite the configurecore instead of this configureservice, and this method body is about configureproxy

Reply:
I understand this `proxyOptions` is a special case, it's applied to `identityClientOptions`, although it is a configuration for proxy, the `configureProxy` method in the factory is actually applied to httpClient.

Reply:
I understand this `proxyOptions` is a special case, it's applied to `identityClientOptions`, although it is a configuration for proxy, the `configureProxy` method in the factory is actually applied to httpClient.

Reply:
let's discuss this offline